ginga/rv/plugins/Catalogs.py

```diff
@@ -372,26 +401,38 @@ def redo(self):
             return True
 
         # Copy the image parameters out to the widget
-        d = { 'ra': ra_ctr, 'dec': dec_ctr, 'width': str(wd),
-              'height': ht, 'r': radius, 'r2': radius,
-              'r1': 0.0,
-              }
+        d = {'ra': ra_ctr, 'dec': dec_ctr, 'width': str(wd),
+             'height': ht, 'r': radius, 'r2': radius,
+             'r1': 0.0}
         self._update_widgets(d)
         return True
 
-    def btndown(self, canvas, event, data_x, data_y):
+    def btndown(self, canvas, event, data_x, data_y, viewer):
         return True
 
-    def btnup(self, canvas, event, data_x, data_y):
-
-        objs = self.canvas.get_items_at(data_x, data_y)
+    def btnup(self, canvas, event, data_x, data_y, viewer):
+        try:
+            objs = self.canvas.get_items_at(data_x, data_y)
+        except Exception:
```

Comment:
Seems like we would want to know about any error here though.  As far as I know there shouldn't be any, so if there is an exception it would be good to know what it is.  Maybe a warning, at least?